strands_navigation_msgs/msg/Edge.msg

```diff
@@ -6,9 +6,8 @@ string map_2d
 bool use_default_recovery
 string[] monitor_recovery_pairs_names
 strands_navigation_msgs/DynClassLoaderDef[] monitor_states
-strands_navigation_msgs/DynClassLoaderDef[] recovery_state_machines
 bool use_default_nav_recovery
```

Comment:
well, no but it would make the message a lot simpler, well is just a thought I'll leave it like this for now and look into it afterwards